risc0/zkp/src/adapter.rs

```diff
@@ -210,45 +212,74 @@ impl PolyExtStep {
         match self {
             PolyExtStep::Const(value) => {
                 let elem = F::Elem::from_u64(*value as u64);
+                tracing::trace!("[{}] {self:?} -> {elem:?}", fp_vars.len());
                 fp_vars.push(F::ExtElem::from_subfield(&elem));
             }
             PolyExtStep::Get(tap) => {
-                fp_vars.push(u[*tap]);
+                let val = u[*tap];
+                tracing::trace!("[{}] {self:?} -> {val:?}", fp_vars.len());
+                fp_vars.push(val);
             }
             PolyExtStep::GetGlobal(base, offset) => {
-                fp_vars.push(F::ExtElem::from_subfield(&args[*base][*offset]));
+                let val = F::ExtElem::from_subfield(&args[*base][*offset]);
+                tracing::trace!("[{}] {self:?} -> {val:?}", fp_vars.len());
+                fp_vars.push(val);
             }
             PolyExtStep::Add(x1, x2) => {
-                fp_vars.push(fp_vars[*x1] + fp_vars[*x2]);
+                let val = fp_vars[*x1] + fp_vars[*x2];
+                tracing::trace!("[{}] {self:?} -> {val:?}", fp_vars.len());
+                fp_vars.push(val);
             }
             PolyExtStep::Sub(x1, x2) => {
-                fp_vars.push(fp_vars[*x1] - fp_vars[*x2]);
+                let val = fp_vars[*x1] - fp_vars[*x2];
+                tracing::trace!("[{}] {self:?} -> {val:?}", fp_vars.len());
```

Comment:
If these aren't needed anymore, let's comment them out.

Reply:
I put them behind a const so they're easy to re-enable.  Does that work?